### PR TITLE
GEOMESA-2016 JDBC SimpleFeatureConverter

### PR DESCRIPTION
**Pull request by elahrvivaz**
* API change: ToSimpleFeatureConverter.fromInputType now returns an Iterator instead of a Seq

Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

Note: stacks on top of #1774

### CODE REVIEW

**Comment by jnh5y** [comment]
No changes to cqs.tsv?

**Comment by elahrvivaz** [comment]
nope, it just uses the built in java jdbc api. at runtime, you have to provide the appropriate driver jar for your database

File: docs/user/accumulo/commandline_tools.rst [hunk 1]
```diff
@@ -168,12 +168,12 @@ If there is whitespace within a keyword, enclose it in quotes for proper functio
       -i instance -z zoo1,zoo2,zoo3 \
       -c catalog -f featureTypeName
 
-get-names
-~~~~~~~~~
+get-type-names
```
Comment (jnh5y) [inline docs/user/accumulo/commandline_tools.rst]: Did all the instances of 'get-names' get updated?

File: docs/user/accumulo/commandline_tools.rst [hunk 2]
```diff
@@ -376,6 +376,13 @@ Using the SFT and Converter config files we can then ingest our csv file with th
 
 For more documentation on converter configuration, see :doc:`/user/convert/index`.
 
+In addition to specifying input files on the command line, you may also pipe data using `stdin`. Note that this will
+only work in local mode, and will only use a single thread for ingestion. Progress indicators may not be entirely
+accurate as the total size isn't known up front. For example::
+
+    $ cat example.csv | geomesa ingest -u username -p password \
+      -c geomesa_catalog -i instance -s /tmp/renegades.sft
```
Comment (jnh5y) [inline docs/user/accumulo/commandline_tools.rst]: Is this bit in the other PR?

File: docs/user/accumulo/commandline_tools.rst (hunk 1 above)
Comment (elahrvivaz) [reply docs/user/accumulo/commandline_tools.rst]: i think so... if not here then in the docs pr. might have missed some of course...

File: docs/user/accumulo/commandline_tools.rst (hunk 2 above)
Comment (elahrvivaz) [reply docs/user/accumulo/commandline_tools.rst]: pretty sure it is, but i'll double check when i fix conflicts

**Pull request merged**